README.md

```diff
@@ -1,63 +1,160 @@
 
-1. Run `npm update && npm install`.
-2. Run `npm run build` to compile all contracts.
+- Set up your system.
+- Deploy and test the apps.
 
-## Examples
+Note: You may see example folders in this repo that are not described below. They are either placeholders for future apps, such as the `temp` folder, or they are dapps in progress and we'll add a description when they're finished.
 
-To run any example on your local machine first run `node scripts/createLocal` on a separate terminal.
+## Prerequisites
 
-All examples use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` as the user that deploys and runs all tests. This address is funded by default when creating a local set of chains. Make sure it is funded on testnets as well. Run `node/printBalances` (optionally specify an address if you wish to check an address other than the one above) to check the balances on all five supported testnets of any address.
+1. You'll need to have node.js installed to run network dapps. To make sure you have it installed, run `node -v`. If no version is returned, see [Nodejs.org/downloads](https://nodejs.org/en/download/).
 
-## Deployment
+2. Clone the repo with `git clone https://github.com/axelarnetwork/axelar-local-gmp-examples.git` and `cd axelar-local-gmp-examples`.
 
-Some examples need to first be deployed on the network you wish to run them on. To do this simply run `node scripts/deploy examples/${name} ${local|testnet}`. Replace `${var}` appropriately. Cannot be run for the `sent-token` and `deposit-address` examples as no deployment is needed.
+3. Build contracts and tests (The update and install takes a few minutes.):
+   ```
+   npm update && npm install
+   npm run build
+   ```
 
-## Testing
+3. To connect to the axelar network, run `npm install axelarnetwork/axelar-local-dev`.
 
-All examples can be tested with `node scripts/test/{example-name} ${local|testnet} ...params`. See below for what values params can take. All params are optional and have default values and can be ommited.
+4. To run a local node, open `axelar-local-gmp-examples` in a separate terminal and run `node scripts/createLocal`. You’ll need to have this node running to deploy the dapps.
 
-### Sent token
+   The five supported networks are deployed: Moonbeam (ChainID: 2500), Avalanche (chainID: 2501), Fantom (chainID: 2502), Ethereum (chainID: 2503), and Polygon (chainID: 2504) on port 8500.
 
-Run `node scripts/test examples/send-token ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+5. To make sure that the address we use for examples is funded on all five supported testnets, run `node scripts/checkBalances`. We use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` to deploy and test all examples. Alternatively, you can use `--address [<address>]` to specify a different address.
```

Comment:
We have an issue in for this step that we'll need to look for a resolution for. Would be good to locate the issue #.

Reply:
Issue is here https://github.com/axelarnetwork/axelar-local-gmp-examples/issues/23